tests/network/vmi_multus.go

```diff
@@ -521,7 +521,6 @@ var _ = Describe(SIG("Multus", Serial, decorators.Multus, func() {
 		Context("Security", func() {
 			BeforeEach(func() {
 				const (
-					bridge11CNIType       = "cnv-bridge"
```

Comment:
It was dropped by https://github.com/kubevirt/kubevirt/pull/15583/commits/4810df78d3f49cc951c1757ac0ddfde6bd547df3